language-plutus-core/stdlib/Language/PlutusCore/StdLib/Data/Tuple.hs

```diff
@@ -0,0 +1,101 @@
+-- | @tuple@s of various sizees and related functions.
+
+{-# LANGUAGE OverloadedStrings #-}
+
+module Language.PlutusCore.StdLib.Data.Tuple
```

Comment:
I'd like to keep the internal stdlib modules purely PLCish. Can we move this to `Language.PlutusCore.StdLib.Meta.Data.Tuple` or the like? Maybe even `Language.PlutusCore.Tuple`? I mean, those functions are library functions, but for Haskell users, not PLC users.

Reply:
Well, we do have `fixN` in `Language.PlutusCore.Stdlib.Data.Function`. Probably that should go in `Language.PlutusCore.Stdlib.Data.Function` as well?

Reply:
I'm also not sure how to include those nicely in the testing - at the moment I've arbitrarily put in instantiations at 2 (like we do for `fixN`)